packages/docusaurus-plugin-content-blog/src/authors.ts

```diff
@@ -196,5 +202,33 @@ Don't mix 'authors' with other existing 'author_*' front matter. Choose one or t
     return [authorLegacy];
   }
 
+  const inlineAuthors = updatedAuthors.filter((author) => author.inline);
+
+  const duplicateList = updatedAuthors.filter(
+    (author, index, self) =>
+      index !== self.findIndex((t) => t.name === author.name),
+  );
+
+  // TODO need title check otherwise reports weird cases
+  if (inlineAuthors.length > 0 && params.frontMatter.title) {
+    logger.warn(
+      `Inline authors found in blog [${
+        params.frontMatter.title
+      }] ${inlineAuthors.map((author) => author.name).join(', ')}`,
+    );
+  }
+
+  // TODO need title check otherwise reports weird cases
+  if (duplicateList.length > 0 && params.frontMatter.title) {
```

Comment:
`&& params.frontMatter.title` is wrong, we don't want to ignore blog posts providing title through markdown `# heading`